### PR TITLE
Change the max file size to 250MB; bug 1151599

### PR DESCRIPTION
**Pull request by clouserw**

### CODE REVIEW

File: mkt/constants/submit.py [hunk 1]
```diff
@@ -16,4 +16,4 @@
     (700, 1050, 'full'),  # Because it's proportional, that's why.
 ]
 
-MAX_PACKAGED_APP_SIZE = 100 * 1024 * 1024  # 100MB
+MAX_PACKAGED_APP_SIZE = 250 * 1024 * 1024  # 100MB
```
Comment (chuckharmston) [inline mkt/constants/submit.py]: Update comment?

Comment (mstriemer) [inline mkt/constants/submit.py]: Could assign an `MB` name instead, too.

```py
MB = 1024 * 1024
MAX_PACKAGED_APP_SIZE = 250 * MB

**Comment by mstriemer** [comment]
r+wc

File: mkt/constants/submit.py (hunk 1 above)
Comment (clouserw) [inline mkt/constants/submit.py]: wow, totally missed the comment

**Comment by clouserw** [comment]
Bah, I wanted to do this via github, but two commits for this is ridiculous.  One sec.

**Pull request closed without merging**